backend/open_webui/routers/users.py

```
@@ -162,6 +247,23 @@ async def get_user_info_by_session_user(user=Depends(get_verified_user)):
         )
 
 
+############################
+# GetUserRoleBySessionUser
+############################
+
+
+@router.get("/user/role", response_model=Optional[str])
+async def get_user_info_by_session_user(user=Depends(get_current_user)):
```

Ruff (0.12.2)

Redefinition of unused `get_user_info_by_session_user` from line 239

(F811)